website/static/js/fangorn.js

```diff
@@ -1013,6 +996,491 @@ function scrollToFile(fileID) {
     }
 }
 
+// A fangorn-styled button; addons can reuse this
+var FGButton = {
+    controller: function(args) {
+        var noop = function() {};
```

Comment:
This isn't used here. I suggest allowing args.onclick to default to noop in the view to prevent an error if `onclick` isn't passed.

```js
var onclick = args.onclick || noop;
```